deployment/v3/testrig/uitestrig/install.sh

```
@@ -44,25 +44,88 @@ function installing_uitestrig() {
+    if [ -z "$injiWebUi" ]; then
+       echo "ERROR: injiWebUi url cannot be empty; EXITING;";
+       exit 1;
+    fi
+
+  read -p "Please enter the TEST_URL : " TEST_URL
+    if [ -z "$TEST_URL" ]; then
+       echo "ERROR: Test url cannot be empty; EXITING;";
+       exit 1;
+    fi
+
+  read -p "Please enter the MOSIP_INJIWEB_GOOGLE_REFRESH_TOKEN : " token
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)